packages/reactive-element/src/decorators/base.ts

```diff
@@ -17,3 +17,10 @@ export type Constructor<T> = {
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   new (...args: any[]): T;
 };
+
+export const descriptorDefaults = {
```

Comment:
I don't really want to commit to these exports being in the public API. Can you mark `@internal`? And can these be renamed by Terser?

Reply:
Marked as @internal

Looks like it's not renamed, Terser emits:

```js
const e=(e,t,c)=>(c.configurable=!0,c.enumerable=!0,Reflect.decorate&&"object"!=typeof t&&Object.defineProperty(e,t,c),c);export{e as desc};
```

Gave it a short name since it's not minified, and marked it as internal